ansible/playbooks/roles/opensearch/tasks/configure-ops.yml

```diff
@@ -90,55 +90,46 @@
         {%- if not loop.last -%},{%- else -%}]{%- endif -%}
       {%- endfor -%}
     nodes_dn: >-
-      {{ existing_es_config['opendistro_security.nodes_dn'] if (is_upgrade_run) else
+      {{ existing_es_config['opensearch_security.nodes_dn'] if (is_upgrade_run) else
          _epiphany_nodes_dn }}
-    opendistro_security_allow_unsafe_democertificates: "{{ certificates.files.demo.opendistro_security.allow_unsafe_democertificates }}"
+    opensearch_security_allow_unsafe_democertificates: "{{ certificates.files.demo.opensearch_security.allow_unsafe_democertificates }}"
 
     http_port:      "{{ is_upgrade_run | ternary(existing_es_config['http.port'],      ports.http) }}"
     transport_port: "{{ is_upgrade_run | ternary(existing_es_config['transport.port'], ports.transport) }}"
 
-# When 'opendistro_security.allow_unsafe_democertificates' is set to 'false' all demo certificate files must be removed,
-# otherwise elasticsearch service doesn't start.
+# When 'opensearch_security.allow_unsafe_democertificates' is set to 'false' all demo certificate files must be removed,
+# otherwise opensearch service doesn't start.
 # For apply mode, demo certificate files are removed based only on their names. For upgrade mode,
 # public key fingerprints are checked to protect against unintentional deletion (what takes additional time).
 
 - name: Remove demo certificate files
   include_tasks:
     file: "{{ is_upgrade_run | ternary('remove-known-demo-certs.yml', 'remove-demo-certs.yml') }}"
-  when: not certificates.files.demo.opendistro_security.allow_unsafe_democertificates
+  when: not certificates.files.demo.opensearch_security.allow_unsafe_democertificates
 
-- name: Include log4j patch
-  include_tasks: patch-log4j.yml
+# - name: Include log4j patch
```

Comment:
Why is this commented?

Reply:
Removed as target OPS ver includes log4j patch https://discuss.opendistrocommunity.dev/t/log4j-patch-for-cve-2021-45105/8119